<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QMainWindow" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1082</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TERMOSTATO</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../Downloads/termostato.png</normaloff>../../../../Downloads/termostato.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(104, 83, 140);
font: 75 11pt &quot;Ubuntu Condensed&quot;;
color: rgb(0, 0, 0);</string>
      </property>
      <property name="title">
       <string>Celcios</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLCDNumber" name="outCent">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(81, 118, 133);
color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="inCent">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>99</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(196, 160, 0);
font: 75 11pt &quot;Ubuntu Condensed&quot;;
color: rgb(0, 0, 0);</string>
      </property>
      <property name="title">
       <string>Fharenheit</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLCDNumber" name="outFhar">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(81, 118, 133);
color: rgb(0, 0, 0);

</string>
         </property>
         <property name="value" stdset="0">
          <double>32.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="inFhar">
         <property name="minimum">
          <number>-215</number>
         </property>
         <property name="maximum">
          <number>210</number>
         </property>
         <property name="value">
          <number>35</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;Ubuntu Condensed&quot;;
background-color: rgb(197, 45, 32);
color: rgb(0, 0, 0);
</string>
      </property>
      <property name="title">
       <string>Kelvin</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLCDNumber" name="outKel">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(81, 118, 133);
color: rgb(0, 0, 0);</string>
         </property>
         <property name="value" stdset="0">
          <double>-273.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="inKel">
         <property name="minimum">
          <number>273</number>
         </property>
         <property name="maximum">
          <number>372</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1082</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>inCent</sender>
   <signal>valueChanged(int)</signal>
   <receiver>outCent</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inFhar</sender>
   <signal>valueChanged(int)</signal>
   <receiver>outFhar</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>408</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inKel</sender>
   <signal>valueChanged(int)</signal>
   <receiver>outKel</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>638</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>638</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
